src/ex1/ItemManager.js

```diff
@@ -0,0 +1,60 @@
+class ItemManager{
+    constructor(element) {
+        this.todoList = [];
+        this.pcInstance = new PokemonClient();
```

Comment:
```suggestion
        this.pokemonClient = new PokemonClient();
```
its hard to understand and remember what is "pc". better to use a more clear name